bot/client.js

```diff
@@ -57,9 +57,16 @@ client.on('message', (message) => {
   }
 });
 
-// If User joins server this message will be sent
+// Event listener for new members
 client.on('guildMemberAdd', (member) => {
-  member.sendMessage(`${member.user.username}, Welcome to the Full Sail Armada! These are the terms of service...`);
+  // Send welcome message to members who join
+  member.sendMessage(`${member}, Welcome to the Full Sail Armada! These are the terms of service...`);
```

Comment:
Small detail, but consider lower-casing the "W" in "Welcome" as it comes after a comma